docs/list-of-diagnostics.md

```diff
@@ -134,7 +82,7 @@ if desired.
 | `LOGGEN033` | Method parameter can't be used with a tag provider |
 | `LOGGEN034` | Attribute can't be used in this context |
 | `LOGGEN035` | The logging method parameter leaks sensitive data |
-
+| `LOGGEN036` | A value being logged doesn't have an effective way to be converted into a string |
```

Comment:
This needs to be added to https://aka.ms

Reply:
Can you handle this please?

Reply:
+1

Reply:
Done: https://aka.ms/dotnet-extensions-warnings/LOGGEN036